gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -694,6 +698,8 @@ private void updateOnApiChange(
             isPerfetto ? Messages.CAPTURE_TRACE_PERFETTO : Messages.CAPTURE_TRACE_GRAPHICS);
         withoutBuffering.setEnabled(!isPerfetto);
         withoutBuffering.setSelection(!isPerfetto && trace.getWithoutBuffering());
+        loadValidationLayer.setEnabled(!isPerfetto && getSelectedDevice().isAndroid());
```

Comment:
I think we need to document that this new option doesn't work when doing a perfetto trace. Ideally even deselect/gray the box in the UI to communicate this.

Reply:
Good point, and that's what this line does. Here loadValidationLayer is a `Button`, and `setEnabled()` grays it out when the trace is Perfetto, or the device is not Android.

Reply:
Ah sorry silly me!